src/main/java/sx/blah/discord/api/IDiscordClient.java

```diff
@@ -444,4 +444,27 @@ default void checkLoggedIn(String action) {
 	 * @see <a href=https://discordapp.com/developers/applications/me>Applications</a>
 	 */
 	IUser getApplicationOwner();
+
+	/**
+	 * Gets a list of all categories visible to the bot user on every shard.
+	 *
+	 * @return A list of all visible categories.
+	 */
+	List<ICategory> getCategories();
```

Comment:
I'd argue that this may be completely useless. But whatever

Reply:
I also see no reason for this to exist. Without no other context, categories don't contain useful information.

Reply:
It's consistent with everything else that is an IDiscordObject in IDiscordObject. We retrieve everything that is available on all shards, this really is no different. Maybe some bots may like the statistical utility this can provide, I don't know. But I think it should stay for consistency.